test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/module_args.py

```diff
@@ -34,6 +34,12 @@
 from .utils import CaptureStd, find_executable, get_module_name_from_filename
 
 
+ANSIBLE_MODULE_CONSTURCTOR_ARGS = (
```

Comment:
Does this need to be kept in sync with `module_utils/basic.py`, if so can we import from there, or add a note to remind people to update this list?

Reply:
For some reason I thought it's not really possible to import it, but then I noticed it is... I've changed it to use inspection to retrieve this list in de5218f0603809321cbdf10b5d3377d2ca1c8714.

Reply:
Brilliant, thank you